.github/workflows/frontend.yml

```diff
@@ -7,6 +7,11 @@ jobs:
     steps:    
     - name: Checkout
       uses: actions/checkout@v2
+      
+    - name: Copy images
+      run: |
+        cp -r imgs/logo_name.png frontend/src/
```

Comment:
Not sure if this is the most efficient solution. Why not to load the remote image hosted in GitHub?

Reply:
I'm not sure react can load the image from a remote url